networking_nsxv3/common/config.py

```diff
@@ -149,7 +114,8 @@
     ),
     cfg.StrOpt(
         'nsxv3_login_password',
-        default='VMware1!',
+        default='VMwarensbu_1!',
```

Comment:
I would suggest to leave the default empty.

Reply:
- [x] Remove default password